### PR TITLE
i18n(pt-BR): Updated `reference/directives-reference.mdx`, and `reference/integrations-reference.mdx`

### PR DESCRIPTION
**Pull request by Alynva**
#### What kind of changes does this PR include?

- Translated content

#### Description

- What does this PR change? Give us a brief description.
  - Updated `reference/directives-reference.mdx`, and `reference/integrations-reference.mdx` based on commits in the original pages (see the commit history for more references)

### CODE REVIEW

File: src/content/docs/pt-br/reference/directives-reference.mdx [hunk 1]
```diff
@@ -173,10 +172,15 @@ Se o componente já está escondido e é mostrado por uma media query no seu CSS
 <AlgumComponenteSolid client:only="solid-js" />
 ```
 
+### Diretivas Customizadas do Cliente
+
+Desde o Astro 2.6.0, integrações também podem adicionar diretivas `client:*` customizadas para alterar como e quanto os componentes devem ser hidratados.
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/reference/directives-reference.mdx:177]: A gente se refere me outras páginas como diretivas de cliente, e o "customizada" deve vir depois na frase. Adicionalmente, o "quanto" deveria ser "quando" (when)
```suggestion
### Diretivas de Cliente Customizadas

Desde o Astro 2.6.0, integrações também podem adicionar diretivas `client:*` customizadas para alterar como e quando os componentes devem ser hidratados.
```

File: src/content/docs/pt-br/reference/directives-reference.mdx [hunk 2]
```diff
@@ -173,10 +172,15 @@ Se o componente já está escondido e é mostrado por uma media query no seu CSS
...
+
+Visite a página da [API `addClientDirective`](/pt-br/reference/integrations-reference/#opcao-addclientdirective) para aprender mais sobre criar uma diretiva customizada do cliente.
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/reference/directives-reference.mdx:179]:
```suggestion
Visite a página da [API `addClientDirective`](/pt-br/reference/integrations-reference/#opcao-addclientdirective) para aprender mais sobre como criar uma diretiva de cliente customizada.
```

**Review by Yan-Thomas — COMMENTED** [review]
My suggestions for `directives-reference.mdx`, just saw you are adding `integrations-reference.mdx` as well to fix the link, so I'll come back here later!

File: src/content/docs/pt-br/reference/directives-reference.mdx (hunk 2 above)
Comment (Alynva) [reply src/content/docs/pt-br/reference/directives-reference.mdx]: Resolved with 73feda392f7861dba778502e51ca38b39f2f4bbe

File: src/content/docs/pt-br/reference/integrations-reference.mdx [hunk 3]
```diff
@@ -137,6 +142,48 @@ addWatchFile('/home/usuario/.../minha-configuracao.json');
...
 
+#### Opção `addClientDirective`
+
+<Since v="2.6.0" />
+
+**Tipo:** `(directive:` [`ClientDirectiveConfig`](https://github.com/withastro/astro/blob/00327c213f74627ac9ca1dec774efa5bf71e9375/packages/astro/src/%40types/astro.ts#L1872-L1875) `) => void;`
+
+Adiciona uma [diretiva de cliente customizada](/pt-br/reference/directives-reference/#diretivas-de-cliente-customizadas) para ser usada em arquivos `.astro`.
+
+Note que entrupoitns de diretivas são agrupados somente através do esbuild e devem ser mantidos pequenos para que não deixem a hidratação de componentes mais lenta.
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/reference/integrations-reference.mdx:153]:
```suggestion
Note que entrypoints de diretivas são agrupados somente através do esbuild e devem ser mantidos pequenos para que não deixem a hidratação de componentes mais lenta.
```

File: src/content/docs/pt-br/reference/integrations-reference.mdx [hunk 4]
```diff
@@ -149,12 +196,24 @@ Uma função de callback para injetar rotas em um projeto Astro. Rotas injetadas
...
+```js
+injectRoute({
+  // Usa o padrão de sintaxe do Astro para rotas dinâmicas.
+  pattern: '/subpasta/[dinamico]',
+  // Usa a sintaxe de caminho relativo para uma rota local.
+  entryPoint: './src/pagina-dinamica.astro'
+});
+```
+
+Para uma integração projetada para ser instalada em outros projetos, utilize o nome do seu pacote para referenciar a entrypoint da rota.
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/reference/integrations-reference.mdx:210]: Seguindo a técnica de traduzir para pegar o gênero do artigo, "(o) ponto de entrada"
```suggestion
Para uma integração projetada para ser instalada em outros projetos, utilize o nome do seu pacote para referenciar o entrypoint da rota.
```

**Review by Yan-Thomas — COMMENTED** [review]
Just two small suggestions for `integrations-reference.mdx` and then we're good to go!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**